Sources/LicensePlistCore/Extension/APIKit.extension.swift

```diff
@@ -19,4 +19,17 @@ extension LicensePlistExtension where Base: Session {
             return self.sendSync(request)
         }
     }
+
+    static var shared: Session {
+        Session(adapter: GitHubURLSessionAdapter(configuration: .default))
```

Comment:
Worth noting we were unable to do `Base.init(adapter` here because `Session` doesn't have a required initialiser defined.